tests/checkpoint/test_simple_checkpoint.py

```diff
@@ -3462,3 +3466,47 @@ def test_simple_checkpoint_does_not_pass_dataframes_via_validations_into_checkpo
         match='batch_data found in validations cannot be saved to CheckpointStore "checkpoint_store"',
     ):
         context.add_checkpoint(**checkpoint_config)
+
+
+@pytest.mark.unit
+def test_simple_checkpoint_result_validations_include_rendered_content(
+    titanic_pandas_data_context_stats_enabled_and_expectation_suite_with_one_expectation,
+    sa,
+):
+    context: DataContext = titanic_pandas_data_context_stats_enabled_and_expectation_suite_with_one_expectation
```

Comment:
Is this really a unit test? The context in question uses a bunch of file I/O as part of setup right?

Reply:
This was a mistake. ✅